mwms/monitoring/cadvisor.sls

```diff
@@ -0,0 +1,30 @@
+{% set cadvisor_port = 8080 %}
+
+/usr/local/sbin/cadvisor:
+  file.managed:
+    - source: salt://mwms/monitoring/files/cadvisor
+    - mode: 755
+
+/etc/supervisor/conf.d/cadvisor.conf:
+  file.managed:
+    - contents: |
+        [program:cadvisor]
+        command=/usr/local/sbin/cadvisor -port {{ cadvisor_port }}
```

Comment:
I meant to put the whole arguments for cadvisor into a pillar.

For example:
`command=/usr/local/sbin/cadvisor {{ cadvisor_arguments | join(',') }}`